package.json

```diff
@@ -29,6 +29,7 @@
     "express-init": "^1.1.0",
     "express-rate-limit": "^2.11.0",
     "extend": "^3.0.1",
+    "glob": "^7.1.2",
```

Comment:
is this required?

Reply:
Huh. Interestingly no. Good catch. Must be left over from some iteration of the code.